tapir/coop/migrations/0042_auto_20240319_1242.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.23 on 2024-03-19 11:42
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
```

Comment:
Could all the migrations be merged into a single file? Or there are 9 logically separate changes?

Reply:
I'm not sure to understand how to merge migrations files into one? I make the command "make migrations" and django puts out these single files. If you could tell me how to merge migrations files ....

Reply:
I put them in the PR, so nobody would have to make a migration on the server. And the migrations files are on GitHub also as single files in there. Also we have a bug when making a migrations as I discussed with Théo. It makes all the time a needless files in the account-app. If you want to make your own makemigrations command you'll have to delete this file first.

Reply:
Okay, so I read a little bit about that and I think the approach is to remove all the migration files that you added in `tapir/coop/migration` (from 0042 to including 0050) and run `makemigrations` again. That would re-create the changes in a single file replacing what you currently have (confusing 9 files).

Does it make sense?